File: solutions/pzrxi/01.py

param = input("enter number : ")
try:
	intParam = int(param)
	print("output is : ", end="")
	if intParam % 3 < 1 and intParam % 5 < 1:
		print("FizzBuzz")
	elif intParam % 3 < 1:
		print("Fizz")
	elif intParam % 5 < 1:
		print("Buzz")
	else:
		print(param)
except Exception as e:
	print("error occurs!! - please input only number")

File: solutions/theerapolMan/01.php
<?php

  $input = 30;
  
  if(($input>=1) && ($input<=100)){      //เช็คเงื่อนไขว่าค่าใน input อยู่ระหว่าง 1 - 100 หรือไม่
    if(($input%3==0) && ($input%5==0)){  //เช็คเงื่อนไขว่า input หาร 3 และ 5 ลงตัวทั้งคู่หรือไม่
        echo "FizzBuzz";
    }else if($input%3==0){               //เช็คเงื่อนไขว่า input หาร 3 ลงตัว
        echo "Fizz";
    }else if($input%5==0){               //เช็คเงื่อนไขว่า input หาร 5 ลงตัว
        echo "Buzz";
    }else{
        echo $input;
    }
  }else{
        echo $input;
  }

  
?>